README.md

```diff
@@ -1,12 +1,4 @@
-![Streisand Logo](https://raw.githubusercontent.com/jlund/streisand/master/logo.jpg "Automate the effect")
-
-- - -
-[English](README.md), [Français](README-fr.md), | [Mirror](https://area51.threeletter.agency)
-- - -
-
-[![Build Status](https://travis-ci.org/jlund/streisand.svg?branch=master)](https://travis-ci.org/jlund/streisand)
-
-
+![Streisand Logo](https://raw.githubusercontent.com/jlund/streisand/master/logo.jpg "Automate the effect") - - - [English](README.md), [Français](README-fr.md), | [Mirror](https://area51.threeletter.agency) - - - [![Build Status](https://travis-ci.org/jlund/streisand.svg?branch=master)](https://travis-ci.org/jlund/streisand) 
```

Comment:
Oops! Not sure how I munged that up. My intention was to only have the translations & the mirror link on the same line. Fixed in 71f3ecf.

Alternatively I could move the mirror link to its own line between translations & the build status. I kind of liked it next to the translations to take up less vertical space before the "What the heck is Streisand" stuff.